go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/hashicorp/hcl/v2 v2.3.0 // indirect
 	github.com/hashicorp/terraform-config-inspect v0.0.0-20191212124732-c6ae6269b9d7 // indirect
 	github.com/hashicorp/terraform-plugin-sdk v1.7.0
-	github.com/shurcooL/githubv4 v0.0.0-20201206200315-234843c633fa
+	github.com/shurcooL/githubv4 v0.0.0-20210725200734-83ba7b4c9228
```

Comment:
Can we get some insight into why this dependency is changing?  Not a blocker, but hoping to understand what we're currently missing.

Reply:
@jcudit The structs/variables for Conversation Resolution do not exist with the older version of `githubv4`.